enzy_htp/structure/structure.py

```diff
@@ -614,6 +614,48 @@ def assign_ncaa_chargespin(self, net_charge_mapper: Dict[str, Tuple[int, int]]):
 
     #endregion
 
+    def get_residue(self, key:str) -> Residue:
```

Comment:
A function that does the same is `find_residue_with_key()`. Check it out. It takes a tuple as arg tho. We can either make a if flow inside of it or a dispatch.

Reply:
I merged it to get()